service-deployment/src/gpu-exporter/dockerfile

```diff
@@ -29,10 +29,8 @@ RUN apt-get update && \
 
 WORKDIR /go
 
-COPY copied_file/app.go /usr/local/
-COPY start.sh /usr/local/start.sh
+COPY copied_file/exporter/app.go /usr/local/
 RUN go build -v -o /usr/local/app /usr/local/app.go
-RUN chmod u+x /usr/local/start.sh
 RUN chmod u+x /usr/local/app 
-CMD [ "/usr/local/./start.sh" ]
+CMD [ "/usr/local/app" ]
```

Comment:
>ocal/app" [](start = 13, length = 9)

in this case, no need to have start.sh in this folder, right?

Reply:
Yes, no need, i have removed it and test

Reply:
ok  i will fix this